src/App.js

```diff
@@ -1,19 +1,20 @@
-import React from 'react';
-import HomePage from './components/Home/HomePage';
-import './App.css';
-import Footer from './components/Footer/footer';
+import React from "react";
+import { Route, Routes } from "react-router-dom";
+import JobSeekersSignUp from "./components/JobSeekersPage/JobSeekersSignUp";
+import JobSeekersLogin from "./components/LoginPage/Login";
+import CreateAccount from "./components/SignUp/SignUp";
+import HomePage from "./components/Home/HomePage";
+import "./App.css";
+
 
 function App() {
   return (
-    
-    <React.Fragment>
-      
-      <main className='container'>
-        <HomePage/>
-      </main>
-      {/* <Footer/> */}
-      </React.Fragment>
-    
+    <Routes>
+      <Route exact path="/" element={<HomePage />} />
+      <Route path="/jobSeekersSignUp" element={<JobSeekersSignUp />} />
```

Comment:
All router paths should be in lowercase. If there's more than one word, use the `-` separator.
For example: `/job-seekers-signup`